src_c/font.c

```diff
@@ -60,11 +60,15 @@ static PyObject *
 PyFont_New(TTF_Font *);
 #define PyFont_Check(x) ((x)->ob_type == &PyFont_Type)
 
-static int font_initialized = 0;
 static unsigned int current_ttf_generation = 0;
-static const char font_defaultname[] = "freesansbold.ttf";
+#if defined(BUILD_STATIC)
+static int font_initialized = 1;
```

Comment:
why is this change being made here? Why is this initialised at the beginning, even before the init function is called?

Reply:
yeah far from obvious that because SDL_Init + TTF_Init()  are made in main before CPython process the module inittab. so the emscripten handler knows it will use SDL2 next cycle.
i'll put a comment